coverage/src/main/scala/com/databricks/labs/remorph/coverage/QueryRunner.scala

```diff
@@ -21,17 +21,25 @@ abstract class BaseQueryRunner(transpiler: Transpiler) extends QueryRunner {
   override def runQuery(exampleQuery: ExampleQuery): ReportEntryReport = {
     transpiler.transpile(SourceCode(exampleQuery.query)) match {
       case Result.Failure(PARSE, errorJson) => ReportEntryReport(statements = 1, parsing_error = Some(errorJson))
-      case Result.Failure(_, errorJson) => ReportEntryReport(statements = 1, transpilation_error = Some(errorJson))
+      case Result.Failure(_, errorJson) =>
+        // If we got past the PARSE stage, then remember to record that we parsed it correctly
+        ReportEntryReport(parsed = 1, statements = 1, transpilation_error = Some(errorJson))
       case Result.Success(output) =>
         if (exampleQuery.expectedTranslation.map(format).exists(_ != format(output))) {
           val expected = exampleQuery.expectedTranslation.getOrElse("")
           ReportEntryReport(
             parsed = 1,
             transpiled = 1,
             statements = 1,
-            transpilation_error = Some(compareQueries(expected, output)))
+            reconciliation_error = Some(compareQueries(expected, output)))
```

Comment:
our downstream ETL pipelines rely on `transpilation_error` for this purpose. why do we need this? this is still a transpilation error if it's not converting things correctly.

Reply:
Valentin and I wanted to separated the cases where the generation was successful but it does not match the expected output as at the moment we don't get to be told that. However, we can still count this as a transpilation error if you think that downstream tools might need to see it that way.